install.bat

```diff
@@ -0,0 +1,50 @@
+@ECHO off
+
+ECHO "Welcome to the setup for mentorship-backend repo."
+
+ECHO "This installation requires Python version above 3.0."
+
+ECHO "Cloning the repository"
+git clone https://github.com/systers/mentorship-backend.git
+cd mentorship-backend
+
+ECHO "Installing the virtual environment - 'venv'"
+pip3 install virtualenv
+virtualenv venv
+
+CALL venv\Scripts\activate	
+
+ECHO "Activated virtual environment"
```

Comment:
Can you also please show the environment name and how to activate it after the setup finishes?